tests/e2e/dsc_cfmap_deletion_test.go

```diff
@@ -26,99 +23,81 @@ func cfgMapDeletionTestSuite(t *testing.T) {
 	defer removeDeletionConfigMap(testCtx)
 
 	t.Run(testCtx.testDsc.Name, func(t *testing.T) {
-		t.Run("create data science cluster", func(t *testing.T) {
-			err = testCtx.testDSCCreation()
-			require.NoError(t, err, "Error to create DSC instance")
+		// t.Run("create data science cluster", func(t *testing.T) {
```

Comment:
Probably no need to commit commented code?

Reply:
sure, i will clean up these